Helper/Asset/ProcessHelper.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace EMS\ClientHelperBundle\Helper\Asset;
+
+use EMS\ClientHelperBundle\Helper\Elasticsearch\ClientRequest;
+use EMS\ClientHelperBundle\Helper\Elasticsearch\ClientRequestManager;
+use EMS\CommonBundle\Storage\Processor\Config;
+use EMS\CommonBundle\Storage\Processor\Processor;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Twig\Extension\RuntimeExtensionInterface;
+
+class ProcessHelper implements RuntimeExtensionInterface
+{
+    /** @var Processor */
+    private $processor;
+    /** @var ClientRequest */
+    private $clientRequest;
+    /** @var UrlGeneratorInterface */
+    private $urlGenerator;
+
+    public function __construct(Processor $processor, ClientRequestManager $clientRequestManager, UrlGeneratorInterface $urlGenerator)
+    {
+        $this->processor = $processor;
+        $this->clientRequest = $clientRequestManager->getDefault();
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function process(Request $request, string $processor, string $assetHash, string $configHash = null): Response
+    {
+        if ($configHash) {
+            //already generated by twig function!
```

Comment:
dead code

Reply:
if that is covered, this comment is dead code..

Reply:
True will be removed